src/runtime_src/hip/api/hip_device.cpp

```diff
@@ -146,7 +150,7 @@ hipInit(unsigned int flags)
 }
 
 hipError_t
-hipGetDeviceCount(size_t* count)
+hipGetDeviceCount(int* count)
```

Comment:
I assume int  is required by HIP?

Reply:
Sorry, rebase error. It has been reverted back.